src/main/java/seedu/homerce/logic/parser/ParserUtil.java

```diff
@@ -37,6 +37,8 @@
     public static final String MESSAGE_INVALID_DURATION = "Duration should be a number in hours.";
     public static final String MESSAGE_INVALID_MONTH = "Month should be a number between 1 - 12";
     public static final String MESSAGE_INVALID_YEAR = "Year should be a positive valid number";
+    public static final int VALID_YEAR = 2020;
```

Comment:
Should we fix the `VALID_YEAR `to be 2020? Would this still work when I'm using this after the year 2020?